### DISCUSSION TITLE
Without VPN?

### DISCUSSION THREAD

**Discussion by glyh** [SideStore/SideStore#409] [Q&A] [answered]
The very reason I want sideloading is I want a VPN, and it's blocked in some area. But sidestore requires a VPN which is unreasonable for my case.

**Comment by polymo1** [comment, marked as answer]
Sidestore’s “VPN” only connects back to your device. All you need to do is follow the instructions, and enable the vpn. The vpn does not connect you to any other computer outside of your device.

**Reply by riei2922jww** [reply]
How often does the VPN have to be connected? Because I use my regular VPN connection as well.

**Reply by polymo1** [reply]
Any time you want to install an app or refresh SideStore, or any app installed with it.
